README.md

```diff
@@ -96,7 +96,9 @@ store:
       # packages: VerdaccioPackage
       ## Optional. Name of the `kind` to store token metadata to. Defaults to 'VerdaccioToken'.
       # tokens: VerdaccioToken
-
+      ## Optional. The namespace to store entities in.
+      # namespace: [default]
```

Comment:
All JSON is valid YAML. Based on the TypeScript typing for this field, this should be a string and not an array containing a string?

```suggestion
      # namespace: default
```